### PR TITLE
Add improvement to mass delete limit [IT250]

### PR DESCRIPTION
**Pull request by mlorencik**
Add improvement to mass delete limit [IT250]

### CODE REVIEW

File: public_html/layouts/basic/modules/Vtiger/resources/List.js [hunk 1]
```diff
@@ -1600,6 +1600,9 @@ jQuery.Class("Vtiger_List_Js", {
 							url: target.data('url'),
 							data: self.getSearchParams()
 						}).done(function (data) {
+							if (typeof data == 'string') {
```
Comment (TomaszQr) [inline public_html/layouts/basic/modules/Vtiger/resources/List.js]: ?

File: config/ConfigTemplates.php [hunk 2]
```diff
@@ -547,6 +547,16 @@
 			'description' => 'Max number of exported records',
 			'validation' => '\App\Validator::naturalNumber'
 		],
+		'MAX_MASS_DELETE_RECORDS' => [
+			'default' => 1000,
+			'description' => 'Max number of mass deleted records',
+			'validation' => '\App\Validator::naturalNumber'
```
Comment (rskrzypczak) [inline config/ConfigTemplates.php]:
```suggestion
```

File: config/ConfigTemplates.php [hunk 3]
```diff
@@ -547,6 +547,16 @@
...
+		],
+		'MAX_MASS_TRANSFER_OWNERSHIP_RECORDS' => [
+			'default' => 1000,
+			'description' => 'Max number of transfer ownership records',
+			'validation' => '\App\Validator::naturalNumber'
```
Comment (rskrzypczak) [inline config/ConfigTemplates.php]:
```suggestion
```

File: modules/Settings/Mail/actions/MassDelete.php [hunk 4]
```diff
@@ -32,6 +32,13 @@ public function checkPermission(\App\Request $request)
 	public function process(\App\Request $request)
 	{
 		$recordIds = $this->getRecordsListFromRequest($request);
+		$configMaxMassDelete = App\Config::performance('MAX_MASS_DELETE_RECORDS');
+		if (count($recordIds) > $configMaxMassDelete) {
```
Comment (rskrzypczak) [inline modules/Settings/Mail/actions/MassDelete.php]:
```suggestion
		if (count($recordIds) > \App\Config::performance('MAX_MASS_DELETE_RECORDS', 1000)) {
```

File: config/ConfigTemplates.php [hunk 5]
```diff
@@ -547,6 +547,16 @@
 			'description' => 'Max number of exported records',
 			'validation' => '\App\Validator::naturalNumber'
 		],
+		'MAX_MASS_DELETE_RECORDS' => [
```
Comment (rskrzypczak) [inline config/ConfigTemplates.php]: camelCase -> maxMassDeleteRecords

File: config/ConfigTemplates.php [hunk 6]
```diff
@@ -547,6 +547,16 @@
...
+			'default' => 1000,
+			'description' => 'Max number of mass deleted records',
+			'validation' => '\App\Validator::naturalNumber'
+		],
+		'MAX_MASS_TRANSFER_OWNERSHIP_RECORDS' => [
```
Comment (rskrzypczak) [inline config/ConfigTemplates.php]: camelCase

File: modules/Vtiger/actions/TransferOwnership.php [hunk 7]
```diff
@@ -51,7 +58,7 @@ public function process(\App\Request $request)
 			}
 		}
 		$response = new Vtiger_Response();
-		$response->setResult(true);
+		$response->setResult(['notify' => ['text' => \App\Language::translateArgs('JS_RECORDS_TRANSFERRED_SUCCESSFULLY'), 'type' => 'info']]);
```
Comment (TomaszQr) [inline modules/Vtiger/actions/TransferOwnership.php]: \App\Language::translate('JS_RECORDS_TRANSFERRED_SUCCESSFULLY')

File: modules/Settings/Mail/actions/MassDelete.php (hunk 4 above)
Comment (mlorencik) [reply modules/Settings/Mail/actions/MassDelete.php]: it use two times

**Pull request merged**